service/src/main/kotlin/fi/espoo/evaka/pis/EmployeeQueries.kt

```diff
@@ -24,13 +24,20 @@ data class NewEmployee(
     val lastName: String,
     val email: String?,
     val externalId: ExternalId?,
+    val employeeNumber: String?,
     val roles: Set<UserRole> = setOf()
 )
 
 data class EmployeeUser(
     val id: EmployeeId,
     val firstName: String,
     val lastName: String,
+    val employeeNumber: String?,
```

Comment:
Same here: this type is used for stuff that needs to go to the user session, so the employee number is probably not needed. Note how even the external id is missing, because we don't need that in the session

Reply:
Fixed